### PR TITLE
(Pre)review tweaks.

### PR DESCRIPTION
**Pull request by pq**
Based on the review of `when` a few pre-emptive strikes.

Note that I kept true to the annotate non-private method signatures.  (`_` prefixed methods don't need to declare returning `dynamic` for example.  FWIW: it wouldn't bother me if they did...)

@devoncarew @seaneagan

### CODE REVIEW

**Comment by pq** [comment]
Issue tracking coveralls: #11.

File: pubspec.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ authors:
   Sean Eagan <seaneagan1@gmail.com>
 description: Like unix which(1).  Check for and locate installed executables.
```
Comment (devoncarew) [inline pubspec.yaml]: Perhaps: `Like unix which(1) - check for and locate installed executables.`?

Comment (pq) [inline pubspec.yaml]: Done!

**Comment by devoncarew** [comment]
This is missing a bug reference: https://github.com/dart-lang/which/blob/master/lib/src/util.dart#L10. We should add one or remove the line -

**Comment by devoncarew** [comment]
lgtm

**Comment by seaneagan** [comment]
lgtm

**Pull request merged**